cli/binary/switch.go

```diff
@@ -0,0 +1,147 @@
+package binary
+
+import (
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/apex/log"
+	"github.com/fatih/color"
+	"github.com/manifoldco/promptui"
+	"github.com/tarantool/tt/cli/config"
+	"github.com/tarantool/tt/cli/search"
+	"github.com/tarantool/tt/cli/util"
+	"github.com/tarantool/tt/cli/version"
+)
+
+// SwitchCtx contains information for switch command.
+type SwitchCtx struct {
+	// ProgramName is a program name to switch to.
+	ProgramName string
+	// Version of the program to to switch to.
+	Version string
+}
+
+// ansi is a string to clean other strings from system characters using regexp.
```

Comment:
What does "other strings" mean? What are system characters? Why do we need this cleanup?